bot/cogs/queue.py

```diff
@@ -344,11 +282,13 @@ async def cap(self, ctx, *args):
             except ValueError:
                 embed = discord.Embed(title=f'{new_cap} is not an integer', color=self.bot.color)
             else:
-                if new_cap < 2 or new_cap > 100:
+                if new_cap == capacity:
+                    embed = discord.Embed(title=f'Capacity is already set to {capacity}', color=self.bot.color)
+                elif new_cap < 2 or new_cap > 100:
                     embed = discord.Embed(title='Capacity is outside of valid range', color=self.bot.color)
```

Comment:
Tell the user what the valid range is

Reply:
True. Can do this outside of the PR since it's unrelated.

Reply:
That's a good shout